pyproj/_geod.pyx

```diff
@@ -161,46 +207,147 @@ cdef class Geod:
         double lat2,
         int npts,
         bint radians=False,
+        int initial_idx=1,
+        int terminus_idx=1,
+        object out_lons=None,
+        object out_lats=None,
     ):
         """
         given initial and terminus lat/lon, find npts intermediate points.
+        returns lons, lats buffers
+        """
+        res = \
+            self._inv_or_fwd_intermediate(
```

Comment:
I don't see `_npts` used anywhere. Mind removing it?

Reply:
I thought to leave it for backwards compatibility. 
Is the policy not to maintain backwards compatibility with names that prefixed with `_` ?

Reply:
> Is the policy not to maintain backwards compatibility with names that prefixed with _ ?

If it is prefixed with `_`, it is private should only be used internally by `pyproj` (if someone does use it, it is at their own risk). So, backwards compatibility doesn't need to be maintained.